test/functional/CommandCallFunctional.js

```diff
@@ -18,11 +18,13 @@ describe('CommandCall Functional Tests', function () {
       connection.add(new CommandCall({ command: 'RTVJOBA USRLIBL(?) SYSLIBL(?)', type: 'cl' }));
       connection.run((error, xmlOut) => {
         expect(error).to.equal(null);
-        parseString(xmlOut, (parseError, result) => {
-          expect(parseError).to.equal(null);
-          expect(result.myscript.cmd[0].success[0]).to.include('+++ success RTVJOBA USRLIBL(?) SYSLIBL(?)');
-          done();
-        });
+
+        const parser = new XMLParser();
+        let result;
```

Comment:
Overall, the changes look better but one more nit:

can we just inline the variable initialization?

No need to declare result un-initialized then initialize it in the next line